cosmwasm/lib/.gitignore

```diff
@@ -0,0 +1,9 @@
+lib*.a
+Cargo.lock
+Enclave_t.c
+Enclave_t.h
+Enclave_t.o
+
+Enclave_u.c
+Enclave_u.h
+Enclave_u.o
```

Comment:
Please use more specific .gitignore files in each directory. not each directory has all these files, and `wasmi-runtime` should definitely keep its `Cargo.lock` file.